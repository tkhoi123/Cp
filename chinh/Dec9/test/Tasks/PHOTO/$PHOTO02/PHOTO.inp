9
3 1 5 9 7 2 8 6 4 
3 1 6 5 7 2 9 8 4 
3 1 5 7 8 2 9 6 4 
3 1 2 7 5 9 8 4 6 
7 2 9 8 6 4 3 5 1 
