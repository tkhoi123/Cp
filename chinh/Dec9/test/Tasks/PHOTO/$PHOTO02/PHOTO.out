3 1 5 7 2 9 8 6 4 